src/Microsoft.DotNet.ProjectModel/ProjectDescription.cs

```diff
@@ -43,5 +43,10 @@ public ProjectDescription(string name, string path)
         public Project Project { get; }
 
         public TargetFrameworkInformation TargetFrameworkInfo { get; }
+        
+        public OutputPathCalculator GetOutputPathCalculator()
```

Comment:
Should we add nullable parameters to this API? It may not be clear to users that the runtime and baseOutputPath will always be null when using this one versus the one in ProjectContext.